integration/template/node_operator_charts_values.go

```diff
@@ -0,0 +1,11 @@
+package template
+
+const NodeOperatorChartValues = `
+clusterName: ${CLUSTER_NAME}
+Installation:
+  V1:    
+    Secret:      
+      Registry:
+        PullSecret:
+          DockerConfigJSON: "{\"auths\":{\"quay.io\":{\"auth\":\"$REGISTRY_PULL_SECRET\"}}}"
+`
```

Comment:
Everything for the node operator should already be there and should then not be defined specifically in the `kvm-operator` repo. Further I think the `node-operator` is public and the general tone recently was to get rid of pull secrets for public services. Uncertain though.